tests/ast/test_ast_driver.py

```diff
@@ -226,20 +227,17 @@ def test_ast(session, tables, test_case):
                     "## EXPECTED UNPARSER OUTPUT\n\n",
                     actual.strip(),
                     "\n\n## EXPECTED ENCODED AST\n\n",
-                    base64_str.strip(),
+                    base64_str_to_textproto(base64_str.strip()),
                     "\n",
                 ]
             )
     else:
         try:
             # Protobuf serialization is non-deterministic (cf. https://gist.github.com/kchristidis/39c8b310fd9da43d515c4394c3cd9510)
             # Therefore unparse from base64, and then check equality using deterministic (python) protobuf serialization.
-            actual_message = proto.Request()
-            actual_message.ParseFromString(base64.b64decode(base64_str.strip()))
-
-            expected_message = proto.Request()
-            expected_message.ParseFromString(
-                base64.b64decode(test_case.expected_ast_base64.strip())
+            actual_message = base64_str_to_request(base64_str.strip())
```

Comment:
let's remove (or set to dummy value) fields for 
1.) temp tables
2.) python client version

Reply:
Updated